### DISCUSSION TITLE
linuxgame网站   游戏详情介绍界面评论时会出现二次评论 页面报评论内容为空

### DISCUSSION THREAD

**Discussion by likamao** [Gamuxorg/bbs#211] [游戏或网站运行反馈]
问题描述：linuxgame 游戏站bug

操作系统：deepin

bug描述： vivaldi浏览器打开 在游戏详情介绍界面评论时会出现二次评论 页面报评论内容为空

自行尝试解决bug的方式和结果：再次刷新页面

**Comment by shouhuanxiaoji** [comment]
能方便发个截图么

**Comment by likamao** [comment]
> 能方便发个截图么

现在是 ubuntu 上的firefox 
![Firefox_Screenshot_2021-02-01T01-05-07 941Z](https://user-images.githubusercontent.com/28328475/106404236-2cc12380-646d-11eb-8607-6b32035dd280.png)
![Screenshot from 2021-02-01 09-05-36](https://user-images.githubusercontent.com/28328475/106404251-3480c800-646d-11eb-9faa-79bf5d633b0b.png)

**Comment by likamao** [comment]
> 能方便发个截图么

我在当前评论页面评论完：“我玩了60个小时......"的评论后，接着我**继续**在评论框里评论就会报 图2 的错误。

**Comment by shouhuanxiaoji** [comment]
我看了一下，你第二次评论的时候，评论框里是不是内容是空的啊？如果是空的就会报错。

**Comment by likamao** [comment]
> 我看了一下，你第二次评论的时候，评论框里是不是内容是空的啊？如果是空的就会报错。
不是空的，第二次评论的时候 我写了测试评论四个字，然后点击发布评论，这时，输入框被清空同时弹出错误提示。所以错误提示下评论框是空的。

**Comment by shouhuanxiaoji** [comment]
明白了 我再看一下
